src/templates/jobs_templates/public/fix_nested_jobs/details.html

```
@@ -7,9 +7,9 @@
 <tr>
     <th>File:</th>
     <td>
-        <a href="https://commons.wikimedia.org/wiki/{{ result_data.filename }}" target="_blank"
-            rel="noopener noreferrer">
-            {{ result_data.filename }}
+        <a href="https://commons.wikimedia.org/wiki/File:{{ result_data.filename | replace("File:", "") }}"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)